README.md

```diff
@@ -37,19 +37,24 @@ The steps can be outlined like this:
        #    :fingers_crossed:
        python3 -m pip install -U "sphinx>=7.2.5"
 
-3. Build documentation
+3. Development server
+
+       make livehtml
+
+   This will open a local webserver that autoreloads when changes are made to the website.
```

Comment:
Here I think you should clarify exactly what this means.

Something like:

```
This will open a local webserver that auto-reloads whenever you make changes to the `rst` documentation
or any files in the `docs/` directory.
```
would this be enough, or even more clarification?

Just for the uninitiated.

Reply:
I think this is enough :)